### PR TITLE
fix(flutter_desktop): incorrect colors for switch workspace button

### PR DESCRIPTION
**Pull request by richardshiue**
Before:

|Theme|Light|Dark
|---|---|---|
| Default | Screenshot 2024-09-07 at 6 03 28 PM | Screenshot 2024-09-07 at 6 03 39 PM |
| Dandelion | Screenshot 2024-09-07 at 6 04 12 PM | Screenshot 2024-09-07 at 6 03 56 PM |

After:

|Theme|Light|Dark|
|---|---|---|
|Default| Screenshot 2024-09-07 at 6 02 25 PM | Screenshot 2024-09-07 at 6 01 25 PM |
|Dandelion| Screenshot 2024-09-07 at 5 58 30 PM | Screenshot 2024-09-07 at 5 58 47 PM |

### Feature Preview

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

**Comment by LucasXu0** [comment]
What are the differences between before and after for the default theme mode?

**Comment by richardshiue** [comment]
> What are the differences between before and after for the default theme mode?

Nothing! Just for reassuring that there aren't regressions

File: frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/workspace/sidebar_workspace.dart [hunk 1]
```diff
@@ -227,44 +232,53 @@ class _SidebarSwitchWorkspaceButtonState
...
-                borderRadius: 8.0,
-                figmaLineHeight: 21.0,
-                onSelected: (result) => context.read<UserWorkspaceBloc>().add(
-                      UserWorkspaceEvent.updateWorkspaceIcon(
-                        widget.currentWorkspace.workspaceId,
-                        result.emoji,
+      child: MouseRegion(
+        cursor: SystemMouseCursors.click,
+        child: GestureDetector(
+          onTap: () => _popoverController.show(),
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/workspace/sidebar_workspace.dart:238]:
```suggestion
          onTap: () => _popoverController.show(),
          behavior: HitTestBehavior.opaque,
```

File: frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/workspace/sidebar_workspace.dart [hunk 2]
```diff
@@ -227,44 +232,53 @@ class _SidebarSwitchWorkspaceButtonState
...
-                fontSize: 16,
-                emojiSize: 18,
-                enableEdit: false,
-                borderRadius: 8.0,
-                figmaLineHeight: 21.0,
-                onSelected: (result) => context.read<UserWorkspaceBloc>().add(
-                      UserWorkspaceEvent.updateWorkspaceIcon(
-                        widget.currentWorkspace.workspaceId,
-                        result.emoji,
+      child: MouseRegion(
```
[the remark this answers is not in the document]
Comment (richardshiue) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/workspace/sidebar_workspace.dart]: Done!

**Review by LucasXu0 — APPROVED** [review]

**Comment by LucasXu0** [comment]
@richardshiue It conflicts with the main branch. :)

**Pull request merged**